sawyer_description/urdf/sawyer_base.urdf.xacro

```diff
@@ -379,8 +437,13 @@
     <parent link="right_l5"/>
     <child link="right_l6"/>
     <axis xyz="0 0 1"/>
+<xacro:if value="$(arg gazebo)">
+    <limit effort="9.0" lower="-3.14" upper="3.14" velocity="4.545"/>
```

Comment:
Why change the J6 position limits? That could cause MoveIt trajectories in Gazebo to be different than on the actual robot.

Reply:
Ah yes, forgot to document this. It's due to a how gazebo_ros_control Sim Hardware Interface deals with angles:
https://github.com/ros-simulation/gazebo_ros_pkgs/blob/kinetic-devel/gazebo_ros_control/src/default_robot_hw_sim.cpp#L265-L266

It uses the function [angles::shortest_angular_distance function which limits its answer to be between -pi and pi](https://github.com/ros/angles/blob/master/angles/include/angles/angles.h#L97-L99)

Reply:
I think I may have misread the way the code functions, this is only calculating a delta, and a delta of greater than 2*pi seems not only improbable, but will violate time limits. I will revert this change...

Reply:
I tried this, and both Gazebo and Moveit's trajectories seem to struggle with the simulated robot having such a big range for j6. Other robots have experienced this as well. Keeping the 2*pi range.